cmd/soroban-cli/src/commands/network/container.rs

```diff
@@ -0,0 +1,49 @@
+pub(crate) mod logs;
+mod shared;
+pub(crate) mod start;
+pub(crate) mod stop;
+
+// TODO: remove once `network start` is removed
+pub type StartCmd = start::Cmd;
+// TODO: remove once `network top` is removed
+pub type StopCmd = stop::Cmd;
+
+#[derive(Debug, clap::Subcommand)]
+pub enum Cmd {
+    /// Tail logs of a running network container
+    Logs(logs::Cmd),
+    /// Start network
+    ///
+    /// Start a container running a Stellar node, RPC, API, and friendbot (faucet).
+    ///
+    /// soroban network start <NETWORK> [OPTIONS]
+    ///
+    /// By default, when starting a testnet container, without any optional arguments, it will run the equivalent of the following docker command:
+    /// docker run --rm -p 8000:8000 --name stellar stellar/quickstart:testing --testnet --enable-soroban-rpc
```

Comment:
This is actually out-of-date, we should probably pass `--enable rpc,horizon` or just `--enable rpc`. The command here still works, but it will run everything and we don't need everything.

Reply:
We should probably fix this separately though. Apologies, wasn't meaning to imply it needed fixing here.

Reply:
Nice! I've add a new issue for this: https://github.com/stellar/stellar-cli/issues/1365